webrtc.html

```diff
@@ -4415,8 +4415,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
           <dd>The <code><a>RTCIceTransport</a></code> has received at least one
           remote candidate, and is checking candidate pairs but has not yet
-          found a connection. In addition to checking, it may also still be
-          gathering.</dd>
+          found a connection, or consent checks [[!RFC7675]] have failed on a
```

Comment:
Not a big deal, but it's not 100% clear that the boolean logic is "A and (B or C)" and not "(A and B) or C". Maybe adding "either" would help: "... has received at least remote candidate, and either is checking ..."